modules/cluster-logging-deploy-cli.adoc

```diff
@@ -158,14 +158,14 @@ oc get csv --all-namespaces
 +
 ----
 NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
-default                                                 elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-kube-node-lease                                         elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-kube-public                                             elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-kube-system                                             elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-openshift-apiserver-operator                            elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-openshift-apiserver                                     elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-openshift-authentication-operator                       elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
-openshift-authentication                                elasticsearch-operator.4.5.0-202006160135    Elasticsearch Operator   4.5.0-202006160135               Succeeded
+default                                                 elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+kube-node-lease                                         elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+kube-public                                             elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+kube-system                                             elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-apiserver-operator                            elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-apiserver                                     elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-authentication-operator                       elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-authentication                                elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
```

Comment:
After installing 4.5.2 in cluster-bot, the Cluster Logging and Elasticsearch Operators are:
` clusterlogging.4.5.0-202007012112.p0           Cluster Logging          4.5.0-202007012112.p0              Succeeded`
`elasticsearch-operator.4.5.0-202007012112.p0   Elasticsearch Operator   4.5.0-202007012112.p0              Succeeded`